js/lesson-05.js

```diff
@@ -0,0 +1,125 @@
+// !!!  ДЗ 10. Масив
+// Створити масив, довжину та елементи якого задає користувач.
+// Відсортувати масив за зростанням.
+// Видалити елементи з масиву з 2 по 4 (включно!).
+// У міру змін виводити вміст масиву на сторінку.
+
+// !!! Рішення
+
+// const arrayLength = Number(prompt('Введіть довжину для масиву')); 
+// if (arrayLength === 0) {
+//    alert (`Ви не ввели довжину для масиву`)
+// } else if(isNaN(arrayLength)) {
+//    alert (`Ви ввели не числове значення для довжини масиву`)
+// } else {
+//    const userArray = new Array(arrayLength);
+//    console.log(userArray);
+   
+//    for (let i=0; i<arrayLength; i++) {
+//       userArray[i] = prompt('Введіть дані для масиву','');
+//    };
+//    console.log(`Ви ввели такі данні для масиву: ${userArray}`)
+   
+//    const sortedArray = userArray.sort();
+//    console.log(`Відсортовані данні для масиву: ${sortedArray}`)
+   
+//    const slicedArray = [];
+//    sortedArray.forEach((item, i) => {
+//       if(i < 1 || i > 3) {
+//          slicedArray.push(item);
+//       }
+//    })
```

Comment:
Такий варіант нормальний, але простіше, і на мій погляд правильніше писати так:
```suggestion
   const slicedArray = [...sortedArray];
   slicedArray.splice(1, 3);
```

Reply:
Ми ще не розглядали спред оператор